src/MongoDB/Session.c

```diff
@@ -740,8 +740,10 @@ static HashTable* php_phongo_session_get_debug_info(phongo_compat_object_handler
 	if (intern->client_session) {
 		const mongoc_session_opt_t* cs_opts = mongoc_client_session_get_opts(intern->client_session);
 		ADD_ASSOC_BOOL_EX(&retval, "causalConsistency", mongoc_session_opts_get_causal_consistency(cs_opts));
+		ADD_ASSOC_BOOL_EX(&retval, "snapshot", mongoc_session_opts_get_snapshot(cs_opts));
 	} else {
 		ADD_ASSOC_NULL_EX(&retval, "causalConsistency");
+		ADD_ASSOC_NULL_EX(&retval, "snapshot");
```

Comment:
Note: Session.c doesn't actually have getters for the options passed to `Manager::startSession()`. `Session::getTransactionOptions()` doesn't directly correlate with the `defaultTransactionOptions` session option, since it's (a) the combined options from starting a transaction and merging _those_ options with the session's own defaults and (b) only relevant when a session is in an active transaction.

I forget if we considered adding an API for the `startSession()` options in the past, but I don't feel strongly about it.